src/components/wordLoader/wordLoader.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Icon, Button, TextArea } from "semantic-ui-react";
+
+import { WordLoader } from "./index";
+
+describe("renders an expected Form", () => {
+    /** The "shallow" container for Enzyme to use for our React DOM "parent" */
+    let container;
+
+    /** Render our WordLoader Component for inspection. */
+    beforeAll(() => { container = shallow(<WordLoader />); } );
```

Comment:
This `beforeAll` is a major slowdown. We certainly don't need a new instance of the component for each test, particularly if we're just asserting that it has certain elements as children.